faq: Describe in what condition TiDB and RawKV can be used together (#17828)
pingcap/docs: faq/manage-cluster-faq.md

### Can both TiDB data and RawKV data be stored in the same TiKV cluster?

> It depends on your TiDB version and whether TiKV API V2 ([`storage.api-version = 2`](/tikv-configuration-file.md#api-version-new-in-v610)) is enabled. 

Reviewer: ```suggestion
It depends on your TiDB version and whether TiKV API V2 is enabled ([`storage.api-version = 2`](/tikv-configuration-file.md#api-version-new-in-v610)) . 
```

> It depends on your TiDB version and whether TiKV API V2 is enabled ([`storage.api-version = 2`](/tikv-configuration-file.md#api-version-new-in-v610)) . 

Reviewer: ```suggestion
It depends on your TiDB version and whether TiKV API V2 is enabled ([`storage.api-version = 2`](/tikv-configuration-file.md#api-version-new-in-v610)). 
```

- If your TiDB version is v6.1.0 or later and TiKV API V2 is enabled, TiDB data and RawKV data can be stored in the same TiKV cluster. 

> - Otherwise, the answer is no because the key format of TiDB (or data created using the transactional API) is incompatible with data created using the RawKV API (or data from other RawKV-based services).

Reviewer: ```suggestion
- Otherwise, the answer is no because the key format of TiDB data (or data created using the transactional API) is incompatible with data created using the RawKV API (or data from other RawKV-based services).
```

### Can both TiDB data and RawKV data be stored in the same TiKV cluster?

> Yes, but only when:
> 
> - TiDB version >= `v6.1.0`, and
> - TiKV API V2 is enabled ([`storage.api-version = 2`](/tikv-configuration-file.md#api-version-new-in-v610)).
> 
> Otherwise, no, as key format of TiDB (or data created from the transactional API) is not compatible with data created from RawKV API (or data from other RawKV-based services).

Reviewer: ```suggestion
It depends on your TiDB version and whether TiKV API V2 ([`storage.api-version = 2`](/tikv-configuration-file.md#api-version-new-in-v610)) is enabled. 

- If your TiDB version is v6.1.0 or later and TiKV API V2 is enabled, TiDB data and RawKV data can be stored in the same TiKV cluster. 
- Otherwise, the answer is no because the key format of TiDB (or data created using the transactional API) is incompatible with data created using the RawKV API (or data from other RawKV-based services).
```
